tests/unit/adaptors/graph/test_chains.py

```diff
@@ -234,7 +234,24 @@ def test_short_indirect_import(self):
 
         assert result == {("green.indirect", "purple", "blue.foo")}
 
-    def test_long_indirect_import(self):
+    def test_short_indirect_import_as_module(self):
```

Comment:
I feel we could do with a bit more test coverage in general. Maybe each of the test cases in `TestFindShortestChains` should be parametrized with `as_packages=True/False` and then make sure the graph is constructed so we get different expected in each case. What do you think?

Reply:
Yea I think that sounds good! Adding it in 👍🏼

Reply:
Alright I think I've parametrized everything that's relevant here - all of the tests bar the ones related to long chains